drivers/hmis/lib/tasks/one_time_migration_20241111.rake

```diff
@@ -0,0 +1,53 @@
+desc 'One time data migration to populate created_by and updated_by on Custom Assessments'
+# rails driver:hmis:migrate_assessment_user_references_20241111
```

Comment:
Looks good to me! I successfully ran it locally. It didn't update the majority of my assessments because they are generated so don't have versions, but it did update the ones that were created manually via HMIS, which is expected.